template-processors/applier/Dockerfile.in

```diff
@@ -4,10 +4,10 @@ ENV OPENSHIFT_MODULE_VERSION=0.9.0
 ENV KUBE_MODULE_VERSION=9.0.0
 
 USER root
-RUN PACKAGES=".build-deps gcc musl-dev python-dev libffi-dev openssl-dev libpthread-stubs" \
-  && apk add --no-cache --virtual $PACKAGES \
-  && pip install ansible openshift==$OPENSHIFT_MODULE_VERSION kubernetes==$KUBE_MODULE_VERSION jmespath \
-  && apk del $PACKAGES
+
+RUN yum -y install --disableplugin=subscription-manager python36-devel gcc python3-pip python3-setuptools && \
+    pip3 install ansible openshift==$OPENSHIFT_MODULE_VERSION kubernetes==$KUBE_MODULE_VERSION jmespath && \
+    yum remove -y gcc python36-devel
```

Comment:
Please be consistent with where you're placing the `-y`

Reply:
changes done